client/store/localCart.js

```diff
@@ -0,0 +1,68 @@
+const cartState = {
+  cart: JSON.parse(localStorage.getItem('cart') || '[]')
```

Comment:
Might want to refactor this to not have a cart property and just directly call the or statement, will make future statements clearer